PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+            a = modelweza.replace("SS-NW-", "")
+            b = a.replace("-", "/", 1)
+            poleoznaczenia.insert(1.0, 'TUBES '+strftime('%m %y'+'\n')+numerbezmyslnika+' BNRXXXXX\nTYPE '+b+' WP '+str(int(listaparametrow[9]))+' BAR')
+        else:
+            pass
+    elif cowybrano == 'COLGATE':
+        poleoznaczenia.delete(0.0, END)
+        poleoznaczenia.insert(1.0, strftime('%m.%Y\n')+tu.get()+'\n'+numerZP.get()+'\nwww.tubes-international.com')
+    elif cowybrano == 'PESA (OEM)':
+        poleoznaczenia.delete(0.0, END)
+        poleoznaczenia.insert(1.0, 'Tutaj wpisać oznaczenie PESA')
+
+def szukaj(name1, name2, op):
+    wprowadzony = szukajweza.get()
+    matching = [s for s in listawezy if wprowadzony.lower() in s.lower()]
+    rezultaty.delete(0, END)
+    rezultaty.insert(END, *matching)
+    rezultaty.selection_anchor(0)
+    wprowadzwybor()
+
+def wprowadzwybor():
+    tekstindeksuweza.set(rezultaty.get(ANCHOR))
+
+def kliknijwybor(event):
+    tekstindeksuweza.set(rezultaty.get(ANCHOR))
+
+def wyplujdane():
+    wybranywaz = tekstindeksuweza.get()
+    del listaparametrow[:]
+    for wiersz in range(ws.nrows):
+        if wybranywaz == ws.cell_value(wiersz,0):
+            wierszweza = wiersz
+            for cell in ws.row(wierszweza):
+                cell_value = cell.value
+                if cell.ctype in (2,3) and int(cell_value) == cell_value:
+                    cell_value = str(int(cell_value))
+                elif cell.ctype in (2,3) and str(cell_value) == cell_value:
+                    cell_value = str(cell_value)
+                listaparametrow.append(cell_value)
+
+def wyczyscpola():
+    potwierdzeniewyczyszczenia = messagebox.askyesno('Potwierdź', 'Czy chcesz wyczyścić wszystkie pola formularza?')
+    if potwierdzeniewyczyszczenia:
+        podcisn.activate()
+        badanieTak.select()
+        naccheck()
+        obiektywarunkowklienta= [medium, cisnienieRob, tempWew, tempZew, ilosc, dlugosc, podcisn, cisnienieTest, czasTestu, szukajweza, koncowka1entry, koncowka2entry, entryKlient1, entryKlient2, entryKlient3, numerQC, numerZP]
+        for i in obiektywarunkowklienta:
+            i.delete(0, END)
+            try:
+                i.validate()
+            except AttributeError:
+                pass
+        czypodcisn.set(0)
+        podcisn.disable()
+        tu.delete(0, END)
+        mediumBadania.current([0])
+        numerZam.delete(0, END)
+        tekstindeksuweza.set('')
+        czyzuzycie.set(0)
+        czyodtlu.set(0)
+        czyciaglosc.set(0)
+        poleuwagi.delete('0.0', END)
+        poleoznaczenia.delete('0.0', END)
+        typoznaczenia.current(0)
+        szukajweza.delete(0, END)
+        rezultaty.delete(0, END)
+        tekstindeksuweza.set('')
+        koncowki.current(0)
+        tuleje.current(0)
+        tulejwybor.current(0)
+    else:
+        pass
+
+def generatortolerancji():
+    flaga = listaparametrow[13]
```

Comment:
O tu ładnie widać, gdybyś miał obiekt to odwołałbyś się np tak: `flaga = parameters.toleranceType`